ReadingRainbow/MyBooks.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  ContentView.swift
+//  ReadingRainbow
+//
+//  Created by Tyler Sun on 7/20/23.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct MyBooksReducer: Reducer {
+    
+    struct State: Equatable {
+        @PresentationState var myBookDetail: BookDetailReducer.State?
+        var path = StackState<BookDetailReducer.State>()
```

Comment:
I'm thinking we pull the navigation logic out into a separate View/Reducer. Maybe call it RootView or something and all it needs to do right now is create the NavigationStack and the logic of where to go when each Reducer State is navigated to. That will make it easier to add a View before this one if we decide we need to for some reason (like an onboarding view or maybe a sign in view or something). It also works well with the "Single responsibility" coding concept/rule.